SpatialGDK/Source/SpatialGDKTest/Private/SpatialGDKTestModule.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKTestModule.h"
```

Comment:
I don't think this PR should include this module right now since this is going into master. We still haven't fully decided our testing strategy.